_posts/2022-09-21-hvr-and-hva.markdown

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+font: serif
+title: HVR and HVA
+date: 2022-08-08
+highlight: monokai
+author: Matt Carter, Jack Menzie, Leon Ticharwa
+image: /img/blog/hvr-and-hva/banner.jpg
+tags: hvr hva data replication
+---
+
+# HVR and HVA
+
+## HVR
+
+![Architecture](/img/blog/hvr-and-hva/hvr-architecture.png)
+
+### HVR Hub Machine
+
+### Change Data Capture
+
+HVR and HVA utilise a log-based Change Data Capture (**CDC**) method to replicate data between both heterogeneous and homogeneous databases. A log-based capturing method involves maintaining a transaction log that records the content and metadata of a database, in which only incremental changes are replicated from source to target after the initial synchronisation of data. Through utilisation of a log-based CDC method, the following benefits are achieved:
+
+- Low latency
+- No requirements required for changes to made either programmatically or to the schema of databases
+- Minimal impact to the database itself
+
+These features allow replication to take place between multiple highly distributed systems with impressive speeds, via multiple different replication topologies options.
```

Comment:
Whats impressive? From memory its 250gb per hour of CDC?

Reply:
yea it's better to not be vague. Maybe we could check their website for some data

Reply:
Couldn't find the exact number from the notes and some documentation after looking for a few mins. Will just put it as a conservative 200+ GB per hour based on some of the notes we have